src/main.js

```diff
@@ -5491,13 +5491,19 @@ TimelineStoryteller.prototype.setPlaybackMode = function (isPlayback, addLog) {
     optionDiv.attr("class", "control_div onhover");
 
     d3.select(".introjs-hints").style("opacity", 0);
+    // Set read-only mode for annotation elements in playback mode
+    d3.selectAll(".annotation_control, .annotation_drag_area, .image_drag_area, .caption_drag_area")
+      .style("display", "none");
   } else {
     selectWithParent("#record_scene_btn").attr("class", "img_btn_enabled");
     optionDiv.attr("class", "control_div");
     importDiv.attr("class", "control_div");
     menuDiv.attr("class", "control_div");
 
     d3.select(".introjs-hints").style("opacity", 1);
+    // Remove read-only mode for annotations
+    d3.selectAll(".annotation_control, .annotation_drag_area, .image_drag_area, .caption_drag_area")
+      .style("display", "inline");
```

Comment:
Two things: I wouldn't use `"inline"` as the second parameter here, I would make it `""`.  This will remove the `display` style on these elements. So, say in the css you had set those elements to `display: table`, well now you have to also change this code to set it to `display:table`.  But, if you had it do `""`, it will remove the display style on the elements, and would take the css value of `table`.

Second thing: For readability & less code in general, I would consolidate these into one statement after this if...else.  So, make it
```
if(...) {
...
} else {
...
}
d3.selectAll(".annotation_control, .annotation_drag_area, .image_drag_area, .caption_drag_area")
   .style("display", this.playback_mode ? "none" : "");
```